wiki/Wiki.md

```diff
@@ -0,0 +1,96 @@
+## Table of contents
+
+* [Contributing to the wiki](#contributing-to-the-wiki)
+   * [Opening a pull request](#opening-a-pull-request)
+   * [Failed to push changes to wiki upon PR merge.](#failed-to-push-changes-to-wiki-upon-pr-merge)
+* [Implementation details](#implementation-details)
+   * [Wiki architecture](#wiki-architecture)
+   * [When you make changes through the web interface](#when-you-make-changes-through-the-web-interface)
+   
+
+## Contributing to the wiki
+
+If you notice something about the wiki that could be improved, please let us know! There are a couple of ways you can do so:
+
+* If the improvement you have in mind is big, you want feedback before you start working on it, or you don't have time to make the change yourself, [open an issue](https://github.com/oppia/oppia-android/issues/new) in the source repository.
+* If you can make the change yourself, see the instructions below for opening a pull request (PR).
+
+### Opening a pull request
+
+For your first contribution to oppia-android wiki, you'll need to set up your repository (you only have to do this once):
+
+> **Note**  
+> If you've already set up your repository by following the instructions in the [Installing Oppia Android](https://github.com/oppia/oppia-android/wiki/Installing-Oppia-Android) you can skip this step.
+
+1. [Create a fork](https://github.com/oppia/oppia-android/fork) of the source repository into your user account (which we'll refer to as user from now on).
```

Comment:
Could we just link to the existing set up instructions, instead? Prefer not to duplicate them in case one has to change.

Reply:
Fixed